devfile.yaml

```diff
@@ -1,14 +1,23 @@
-apiVersion: 1.0.0
 metadata:
   name: kubernetes-api
+projects:
+  - name: kubernetes-api
+    source:
+      location: 'https://github.com/devfile/kubernetes-api.git'
+      type: git
+    clonePath: src/github.com/devfile/kubernetes-api
 components:
   - id: ms-vscode/go/latest
     memoryLimit: 512Mi
     preferences:
       go.lintTool: golangci-lint
       go.lintFlags: '--fast'
```

Comment:
I'm not sure if it's related but I see the following warning
![Screenshot_20200611_162421](https://user-images.githubusercontent.com/5887312/84390395-322cdd80-ac00-11ea-8b4f-94e7c4ba7f7a.png)

Reply:
Maybe it's related, but here I have copied the settings of the default Go devfile (from the Dashboard Getting Started). I'd rather fix that (if possible) in a next PR.
Do you agree ?